code/unclassified/leap_year/leap_year.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+
+bool isLeapYear(int year)
+{
+    if (year % 400 == 0)
+        return true;
+
+    if (year % 100 == 0)
+        return false;
+
+    if (year % 4 == 0)
+        return true;
+    return false;
+}
+
+void findLeapYears(int a , int b)
+{
+    cout<<"The leap years between "<<a<<" and "<<b<<" are : \n";
+    for( int i=a ; i<=b ; i++ )
```

Comment:
Remove spaces after and before parentheses, but put them in between the operators.